docs/guide/http_cache.md

```diff
@@ -812,12 +812,12 @@ $response->headers->set('X-Location-Id', '123,212,42');
 
 ### Response tagging in templates
 
-1\. `ez_http_tag_location()`
+1\. `ez_http_cache_tag_location()`
 
 For full content tagging when inline rendering, use the following:
 
 ``` html+twig
-{{ ez_http_tag_location(location) }}
+{{ ez_http_cache_tag_location(location) }}
 ```
 
 2\. `ez_http_tag_relation_ids()` or `ez_http_tag_relation_location_ids()`
```

Comment:
Likely, these functions should also have the `ez_http_cache` prefix

Reply:
@adamwojs 
Could you please confirm that 
`ez_http_tag_relation_ids()` and `ez_http_tag_relation_location_ids()`
should be updated in the docs to 
`ez_http_cache_tag_relation_ids()` and `ez_http_cache_tag_relation_location_ids()`

Reply:
@vidarl `ez_http_cache_tag_location` doesn't exist (yet).And the goal of https://github.com/ezsystems/ezplatform-http-cache/pull/143 wasn't to create it. Could it be done in another PR?

Reply:
@vidarl 

`ez_http_cache_tag_relation_ids` and `ez_http_cache_tag_relation_location_ids`  added in https://github.com/ezsystems/ezplatform-http-cache/pull/143

Doc updated in 4f9b4104442523425640b11a78acf602f611ce6d to have this two new Twig functions with the `ez_http_cache_tag_` prefix.